mediatype/mediatype.go

```diff
@@ -42,7 +42,7 @@ const (
 //   * Newline characters
 func Parse(ctype string) (mtype string, params map[string]string, invalidParams []string, err error) {
 	mtype, params, err = mime.ParseMediaType(
-		fixNewlines(fixUnescapedQuotes(fixUnquotedSpecials(fixMangledMediaType(ctype, ';')))))
+		fixNewlines(fixUnescapedQuotes(fixUnquotedSpecials(fixMangledMediaType(removeTag(ctype), ';')))))
```

Comment:
Should we be consistent with naming and use something like `fixUnexpectedDoctype`?

Reply:
yes, done